packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -80,7 +80,10 @@ class CodePushClientWrapper {
 
   final CodePushClient codePushClient;
 
-  Future<App> createApp({String? appName}) async {
+  Future<App> createApp({
+    String? appName,
+    int? organizationId,
```

Comment:
If the personal organization is exposed, I'd recommend making this required, to make sure we have updated all callers to pass the organization around. 🤷